### ISSUE TITLE
[Bug]: LockScreen authentication error

### ISSUE THREAD

**Issue by HadiAwali** [termux/proot-distro#341]
### Problem description

When I lock Screen or the screen auto locks I put the password of my user and press enter but there is error "authentication error" (Iam sure my user password is 100% correct and I tried to change it but the error still)

### What steps will reproduce the bug?

export XDG_RUNTIME_DIR=${TMPDIR}
termux-x11 :0 >/dev/null &
proot-distro login ubuntu --user user --shared-tmp
export XDG_RUNTIME_DIR=${TMPDIR}
env DISPLAY=:0 cinnamon

### What is the expected behavior?

The LockScreen will not unlock and you should force stop termux and termux-x11 and retype all the commands and not lock your screen again

### Bug report policy acknowledgement

- [X] I understand that PRoot Distro won't fix distribution issues.

**Comment by TomJo2000** [comment]
This doesn't sound like an issue with Proot Distro, but rather with Ubuntu.

It is possible you are experiencing a "silent lockout".
Whereby a password is silently locked out after a specified number of failed attempts, or if it has a specified "expiry date".
Running:
```bash
passwd -u your_username
```
as root (inside the proot distro) should unblock the login.
You may or may not have to set a new password as well using:
```bash
passwd your_username
```

**Comment by HadiAwali** [comment]
> This doesn't sound like an issue with Proot Distro, but rather with Ubuntu.
> 
> It is possible you are experiencing a "silent lockout". Whereby a password is silently locked out after a specified number of failed attempts, or if it has a specified "expiry date". Running:
> 
> ```shell
> passwd -u your_username
> ```
> 
> as root (inside the proot distro) should unblock the login. You may or may not have to set a new password as well using:
> 
> ```shell
> passwd your_username
> ```

I tried and didn't work Iam getting "Incorrect password" error and Iam sure 100% my password is correct and Also tried again to change my password and didnt work and I tested it on debian archlinux and ubuntu same error
